src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -75,25 +89,38 @@ export default function ConfirmedAppointmentListItem({
     },
   );
 
-  const getVideoLocation = () => {
-    if (isAtlas) {
-      return 'at an ATLAS location';
-    }
-    return null;
-  };
-
   let header;
   let location;
-  if (isVideo) {
+  let videoSummary;
+  if (isAtlas) {
```

Comment:
These if statements at the top of this component are pretty intense, might be worth trying to refactor more into separate components.